src/schema/event/resolvers.ts

```diff
@@ -1,7 +1,8 @@
-import Promise from 'promise';
-
+import { QueryResolvers } from '../../types/types';
 import composeQuery from '../../utils/composeQuery';
 import normalizeKeys from '../../utils/normalizeKeys';
+import { EventParams } from './types';
```

Comment:
You can also use generated types `import {QueryEventListArgs} from "../../types/types"`